test/Interop/Cxx/templates/class-template-non-type-parameter.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop)
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop -Xcc -std=c++17)
```

Comment:
It might make sense to (in the distant future, probably) run most of the C++ interop tests with multiple versions of the standard.

Reply:
Agreed.  I wonder if we should be considering C++11, C++14 for the time being.  I think that with the next rebranch, we might be able to add C++17 as it might be complete then.  I do think that is something beyond the scope of this change (not to say that you were asking it to be part of this change).

Not 100% certain I think it would have to be in the distant future - testing against C++11 and C++14 simultaneously at least sounds very reasonable.

Reply:
It should probably be part of adding some other lit improvements around C++ interop. The pattern of `-enable-cxx-interop -I %s/Inputs` is present in almost all C++ interop tests, that could be turned into a lit target, and then we could also specify the standard version.